src/components/prescriptions/PrescriptionGroupComponent.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+    <section class="prescription-container">
+        <div class="prescriptionContainer">
+            <h3 class="prescriptionContainer__header">
+                Dodane recepty
+                <span class="numOfPres">{{ prescriptionsList?.length }}</span>
+            </h3>
+            <ul
+                v-if="prescriptionsList?.length"
+                class="prescriptionContainer__list">
+                <prescription-component
+                    v-for="presc in prescriptionsList"
+                    :key="presc.uid"
+                    :name="presc.doctorData"
+                    :date="presc.date"
+                    :typeofdoc="presc.typeOfDoc"
+                    :medicine="presc.medicineName"
+                    :frequency="presc.frequencyMedicine" />
+            </ul>
+            <div
+                v-else
+                class="fallback">
+                <fallback-info-component :information="testInfo" />
+            </div>
+        </div>
+    </section>
+</template>
+
+<script setup>
+    import { storeToRefs } from 'pinia';
+    import useUserStore from '../../stores/userStore';
+    import { onMounted, computed, ref } from 'vue';
+    import PrescriptionComponent from './PrescriptionComponent.vue';
+    import FallbackInfoComponent from '../FallbackInfoComponent.vue';
+
+    const userStore = useUserStore();
+
+    onMounted(async () => {
+        await userStore.fetchPatientPrescriptions();
+        await userStore.fetchPatientImportPrescriptions();
+    });
+
+    const { getPatientPrescriptions } = storeToRefs(userStore);
+
+    const testInfo = ref('Nie dodano żadnej recepty');
+
+    const prescriptionsList = computed(() => {
```

Comment:
I m not sure but is it redundant?

Reply:
No, if we dont have it in situation when we dont have prescriptions adding by hand, we dont have any comment